templates/gRPC-C++.podspec.template

```diff
@@ -20,108 +20,100 @@
+  )
+  grpcpp_private_headers = filter_grpcpp(
+    set(grpc_all_lib_files("grpc++", ("headers",)))
+    - set(grpc_files)
     # We exclude proto related files in this particular podspec so that we can provide a protobuf-independent lib
-    excl_files = grpcpp_proto_files(filegroups)
-    out = [file for file in out if file not in excl_files]
-
-    out = filter_grpcpp(out)
-    return out
-
-  def grpcpp_public_headers(libs, filegroups):
-    out = grpc_lib_files(libs, ("grpc++",), ("public_headers",))
-    excl_files = grpc_public_headers(libs)
-
+    - set(grpcpp_proto_files)
+  )
+  grpcpp_public_headers = filter_grpcpp(
+    set(grpc_all_lib_files("grpc++", ("public_headers",)))
+    - set(grpc_files)
     # We exclude proto related files in this particular podspec so that we can provide a protobuf-independent lib
-    excl_files += grpcpp_proto_files(filegroups)
-
-    out = [file for file in out if file not in excl_files]
-
-    out = filter_grpcpp(out)
+    - set(grpcpp_proto_files)
+  )
+  grpcpp_abseil_specs = all_grpc_libs_and_abseil_specs("grpc++")[1]
 
-    return out
-
-  def grpc_test_util_files(libs):
-    out = grpc_lib_files(libs, ("grpc_test_util",), ("src", "headers"))
-    return out
-
-  def grpc_test_util_headers(libs):
-    out = grpc_lib_files(libs, ("grpc_test_util",), ("headers",))
-    return out
+  grpc_test_util_files = grpc_all_lib_files("grpc_test_util", ("src", "headers"))
+  grpc_test_util_headers = grpc_all_lib_files("grpc_test_util", ("headers",))
 
   # Tests subspec is currently disabled since the tests currently use `grpc++` include style instead of `grpcpp`.
   # TODO (mxyan): enable Tests subspec after the inclusion style is updated in `test/` directory.
-  def grpcpp_test_util_files(libs, filegroups):
-    out = grpc_lib_files(libs, ("grpc++_test_util",), ("src", "headers"))
-    excl_files = grpc_test_util_files(libs) + grpcpp_private_files(libs, filegroups)
-
+  grpcpp_test_util_files = list(sorted(
+    set(grpc_all_lib_files("grpc++_test_util", ("src", "headers"))) -
+    set(grpc_test_util_files) -
+    set(grpcpp_private_files) -
     # We exclude proto related files in this particular podspec so that we can provide a protobuf-independent lib
-    excl_files += grpc_lib_files(filegroups, ("grpc++_codegen_proto", "grpc++_config_proto"), ("headers", "src"))
-    excl_files += ["test/cpp/util/byte_buffer_proto_helper.cc",
-                   "test/cpp/util/byte_buffer_proto_helper.h",
-                   "test/cpp/end2end/test_service_impl.cc",
-                   "test/cpp/end2end/test_service_impl.h"]
-    excl_files += [file for file in out if file.endswith(".proto")]
-
-    out = [file for file in out if not file in excl_files]
-
+    set(grpcpp_proto_files) -
+    set(["test/cpp/util/byte_buffer_proto_helper.cc",
+         "test/cpp/util/byte_buffer_proto_helper.h",
+         "test/cpp/end2end/test_service_impl.cc",
+         "test/cpp/end2end/test_service_impl.h"]) |
```

Comment:
In the original `grpcpp_test_util_files` there was one line
```
excl_files += [file for file in out if file.endswith(".proto")]
```
Where is this line translated to?

Reply:
Oops I missed that part. Added again.